src/pull-request-handler.js

```diff
@@ -22,7 +22,9 @@ module.exports = async context => {
     while ((match = regex.exec(chunk)) !== null) {
       const parsed = parseChunk({ match, context, config })
 
-      if (parsed.filename === '.github/config.yml') continue
+      if (parsed.filename === '.github/config.yml' || (config.exclude && parsed.filename.test(new RegExp(config.exclude)))) {
```

Comment:
@OmgImAlexis 
I don't think this is going to work, as the correct syntax is `new RegExp(/pattern/).test(string)`.
You can (and maybe should) initialize the exclude pattern outside the loops, then:
```js
if (parsed.filename === '.github/config.yml' || (config.exclude && excludePattern.test(parsed.filename))) {
```

As for tests, a pattern you can use is this:
```js
// starts with (^) "bin/"
/^bin\//
```

```js
const exclude = /^bin\//;
const pattern = new RegExp(/^bin\//);
const excludedFiles = ['bin/testfile.js', 'bin', 'bins/heythere.js', 'src/main.js'].filter(str => pattern.test(str));
// Result: ['bin/testfile.js']
```

Reply:
> I don't think this is going to work, as the correct syntax is

Yep good catch - thats what tests are for 😜 thanks @sharkykh